### DISCUSSION TITLE
Change rounded-full from "9999px" to "50%"

### DISCUSSION THREAD

**Discussion by StevenJPx2** [tailwindlabs/tailwindcss#6788] [Ideas]
If we do this change:

1. Having a relative value will ensure that the element will be rounded always, regardless of its size
2. Animations for a circle to square will not look bad. Currently, if we have a hover state for a button to go from rounded-full to rounded-none, the animation has to transition from 9999px to 0px, which makes it run too fast. Setting it to 50% however, makes it run properly.

**Comment by RobinMalfait** [comment]
Hey! Thank you for your question!
Much appreciated! 🙏

This is something that we will likely not change, but that you can change in your own config if you want this. https://tailwindcss.com/docs/border-radius#customizing-your-theme

The biggest problem with relative percentage based values is for non-square values.
https://play.tailwindcss.com/M7aKDp0xt5

**Reply by StevenJPx2** [reply]
Seems good. Thanks for the reply!
